### PR TITLE
[#371] Document file reupload

### PR DESCRIPTION
**Pull request by filip-kopecky**
Added the possibility to reupload a file in a document.

Closes: #371

### CODE REVIEW

File: src/component/resource/document/ModifyFile.tsx [hunk 1]
```diff
@@ -4,25 +4,32 @@ import IfUserIsEditor from "../../authorization/IfUserIsEditor";
...
 
-  const performAction = (label: string) => {
+  const performAction = (label: string, file?: File) => {
     const modifiedFile = Object.assign({}, props.file, { label: label.trim() });
-    props.performAction(modifiedFile).then(toggle);
+    props
+      .performRename(modifiedFile)
+      .then(() => {
+        if (file) return props.performFileUpdate(props.file, file);
+        else return Promise.resolve();
```
Comment (ledsoft) [inline src/component/resource/document/ModifyFile.tsx]: Please, reformat to blocks enclosed in `{}`.

File: src/i18n/en.ts [hunk 2]
```diff
@@ -182,7 +182,7 @@ const en = {
     "asset.create.hideAdvancedSection": "Hide advanced options",
     "asset.remove.tooltip": "Remove this asset",
     "asset.remove.dialog.title": 'Remove {type} "{label}"?',
-    "asset.rename.dialog.title": 'Rename {type} "{label}"',
+    "asset.rename.dialog.title": 'Modify {type} "{label}"',
```
Comment (ledsoft) [inline src/i18n/en.ts]: This should apparently be called `asset.modify.dialog.title`, as the translation is not showing properly in the UI when switched to English.

Comment (ledsoft) [reply src/i18n/en.ts]: See the corresponding constant in `cs.ts`.

**Review by ledsoft — CHANGES_REQUESTED** [review]
Just minor fixes requested, otherwise works fine.

File: src/i18n/en.ts (hunk 2 above)
Comment (filip-kopecky) [reply src/i18n/en.ts]: Oops, missed that one, sorry

**Comment by filip-kopecky** [comment]
@ledsoft Added a commit which resolves comments.

**Review by ledsoft — APPROVED** [review]
LGTM

**Pull request merged**